pkg/systemd/logsJournal.jsx

```diff
@@ -287,7 +287,6 @@ export class JournalBox extends React.Component {
             /* Show 'Load earlier entries' button if we didn't reach start yet */
             this.state.didntReachStart
                 ? <EmptyStatePanel action={_("Load earlier entries")}
-                             actionInProgressText={_("Loading earlier entries")}
```

Comment:
Thanks for spotting! Can you please fix the capitalization in the commit message? It's useful for grepping component names in the log.